packages/shared/apis/Players.ts

```diff
@@ -47,4 +52,23 @@ export class Players extends ExposableAPI implements IPlayers {
       return { userId }
     })
   }
+
+  @exposeMethod
+  async getPlayersInScene(): Promise<{ userId: string }[]> {
+    const parcelIdentity = this.options.getAPIInstance(ParcelIdentity)
+
+    const result = getInSceneAvatarsUserId(parcelIdentity.cid).map((userId) => {
+      return { userId }
+    })
+
+    // check also for current user, since it will appear in `getInSceneAvatarsUserId` result
+    if (isWorldPositionInsideParcels(parcelIdentity.land.sceneJsonData.scene.parcels, lastPlayerPosition)) {
+      const currentUserId = getCurrentUserId(store.getState())
+      if (currentUserId) {
+        result.push({ userId: currentUserId })
+      }
+    }
```

Comment:
How can we be sure to not be pusing the userId twice? where is the logic that prevents `result` from containing our userId?

Reply:
Renderer won't report current user's avatar since it is only reporting `AvatarShape` and the avatar for current user is not making use of an `AvatarShape`. I could add a check if `userId` is already included if you want

Reply:
Ok sounds good. I think a that would be a good defensive measure 👍